src/Transport/Sendmail.php

```diff
@@ -148,8 +148,14 @@ protected function prepareRecipients(Mail\Message $message)
     {
         $headers = $message->getHeaders();
 
-        if (! $headers->has('to')) {
-            throw new Exception\RuntimeException('Invalid email; contains no "To" header');
+        if (! ($hasTo = $headers->has('to')) && ! $headers->has('cc') && ! $headers->has('bcc')) {
```

Comment:
I got a bit confused with this assignment in `if` statement, in general I like them, but only when we have single condition, here it's more tricky so I would suggest to move assignment before the `if` statement and then use the variable in the statement with other checks:
```php
$hasTo = $headers->has('to');
if (! $hasTo && ! $headers->has('cc') && ! $headers->has('bcc')) {
```

Reply:
done